### DISCUSSION TITLE
Retired Framework is listing in Publish project page of Dock dev creation portal

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-coKreat/Community#94] [Q&A] [answered]
### **Prerequisites**
I have searched for existing issues that already report this problem, without success.

### **Current Behaviour**
I'm trying to remove a framework that created for testing purpose called [TPD 2] in dock dev creation portal. So i've retired the framework in learning-service machine and it's successfully retired but it's still coming in channel Read API [https://dock.sunbirded.org/learner/channel/v1/read/<Channel_ID>]

### **Expected Behaviour**
While we retire the framework it should not list in framework type selection of publish project page. But it's listing. 

### **Steps to Reproduce**
	1. As a creator fill the mandatory fields in Details page
	2. In project scope page, Open the framework type drop down.
	
### **API Endpoints used**
       1. https://dock.sunbirded.org/learner/channel/v1/read/<Channel_ID>
       2. https://dock.sunbirded.org/learner/framework/v1/read/<Framework_ID>
        
### Additional Information
Tested on Google Chrome: Version 114.0.5735.198 (Official Build) (arm64)

can anybody help on this.
@amitpriyadarshi

cc: @kiranharidas187

**Comment by Aparna-karri591** [comment]
Tagging @vaivk369

**Comment by vaivk369** [comment]
We have raised PR, to take live frameworks from the list of frameworks linked to channel.
https://github.com/Sunbird-coKreat/creation-portal/pull/2258/files 
@ramkumar-pacewisdom Let us know if you want to make any enhancement around this.

**Comment by vaivk369** [comment, marked as answer]
@ramkumar-pacewisdom The https://github.com/Sunbird-coKreat/creation-portal/pull/2258/files PR is merged in release-7.0.0 which is ongoing release. The deployment is done on coKreat dev environment. Now you can see only live frameworks linked to channel.

**Reply by ramkumar-ns-dev** [reply]
@vaivk369 Now issue got resolved and working, Thank you.
